tests/utils/logging_tester.py

```diff
@@ -0,0 +1,79 @@
+from functools import wraps
+import os
+import json
+
+
+def _check_logs(reference_logfile, generated_logfile):
+    reference_logs = open(reference_logfile, "r").readlines()
+    generated_logs = open(generated_logfile, "r").readlines()
+
+    if len(reference_logs) != len(generated_logs):
+        msg_assert = ""
+        if len(reference_logs) > len(generated_logs):
+            diff = len(reference_logs) - len(generated_logs)
```

Comment:
Nitpick (non-blocking): lines 13 and 18 are duplicated. They could be moved outside of the `if`